plugin/index.php

```diff
@@ -19,13 +23,26 @@ function gutenberg_menu() {
 	);
 }
 
+// Hook.
+add_action( 'admin_enqueue_scripts', 'gutenberg_scripts_and_styles' );
```

Comment:
It was an intentional decision. See this:
<img width="587" alt="image 2017-03-16 at 6 59 27 pm public" src="https://cloud.githubusercontent.com/assets/960133/23999698/fbdfe2d8-0a7a-11e7-96df-fcdb24577e2f.png">

Having function and file level DocBlocks right after one and another makes less sense. That's why I kept the hooks at the top.